test/test-sets/schema-tests.jl

```diff
@@ -0,0 +1,28 @@
+import JuDOMO: match_domo_types, create_dataset_schema
+import JSON: json
+using Dates
+
+## test sets for matching Julia's types to Domo's
+types = [Int32, Int64, Float32, Float64, String, Dates.Date, Dates.DateTime]
+expected_types = ["LONG", "LONG", "DOUBLE", "DOUBLE", "STRING", "DATE", "DATETIME"]
+
+## test sets for schema creation
+schema_test_mathematicians = Dict(
+    "name" => "Leonhard Euler Party",
+    "description" => "Mathematician guest list.",
+    "rows" => 3,
+    "schema" => Dict(
+        "columns" => [ Dict(
+            "type" => "STRING",
+            "name" => "Friend"
+        ), Dict(
+            "type" => "STRING",
+            "name" => "Attending"
+        ) ]
+    )
+) |> json
```

Comment:
This is based on the quickset docs found here: https://developer.domo.com/docs/dataset/quickstart-4